### PR TITLE
Docs: Remove types from tsdoc

### LINKED ISSUE #214: Remove types from tsdocs
## Description

Remove types from all the existing tsdocs. Tsdoc does not require specifying a type for each function parameter in TypeScript.

## Criteria

- [ ] -

## Notes

-

### PR DESCRIPTION
**Pull request by jaas666**
## Overview

Fixes #214
Remove types from tsdoc.

## Assignee Checklist:

- [x] The naming convention of the PR title is correct (See the comment at the top of this template)
- [x] The base branch is correct (See: [Types of Branches](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#types-of-branches))
- [x] The branch name follows the [Branch Naming Conventions](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/CONTRIBUTING.md#branch-naming-conventions)
- [ ] The correct assignees and reviewers have been designated for this PR
- [x] The coding style follows the [Coding Style Guide](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [x] All the related issues are associated with this PR
- [x] All criteria in the associated issues are met (please tick the checkboxes)
- [x] My changes do not generate new warnings or errors (especially in the console)

## Reviewer Checklist:

- [x] Double-check the "Assignee Checklist"
- [x] The code follows the generic best practices and our [Coding Style Guide](https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md)
- [x] The code is well-commented and easy to understand
- [x] The UI changes accurately reflect the provided design

### CODE REVIEW

File: src/middleware.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { NextRequest, NextResponse } from 'next/server';
 
 /**
  * Next.js middleware is the code which is run before a request is completed
- * @param request: NextRequest
+ * @param NextRequest
```
Comment (nick-y-ito) [inline src/middleware.ts:8]: This notation would be better to match others.
```ts
@param request - NextRequest
```

**Review by nick-y-ito — CHANGES_REQUESTED** [review]
@jaas666 
Thank you for your contributions to Nishiki! I kindly request that you review the following points:
- [ ] The point I commented on
- [ ] There are four test files that still have types in their TSDocs

File: src/middleware.ts (hunk 1 above)
Comment (jaas666) [reply src/middleware.ts]: `@params` used throughout the codebase aren't using the dash `-`.
Looks like `@properties` are the ones using the dash.

Do you want to change all params to use the dash too?

Comment (nick-y-ito) [reply src/middleware.ts]: @jaas666 
Thank you for bringing that to my attention! I appreciate it.

I looked it up in the [official documentation](https://tsdoc.org/pages/tags/param/) and noticed that it uses dashes. Thus, we kindly request updating everything using `-`.

Comment (jaas666) [reply src/middleware.ts]: Done 👍

Comment (nick-y-ito) [reply src/middleware.ts]: It seems that there are multiple parts that still remain without a dash. Could you please search for `@param` and fix all of them? Thank you so much!

Comment (jaas666) [reply src/middleware.ts]: I'm only seeing without a dash the ones that don't have a description. Do you want to add the dash even though there's no description?
For example, do you want to change `@param event` to `@param event -`

Comment (nick-y-ito) [reply src/middleware.ts]: I am sorry, I was misunderstood. You are totally right!

Comment (jaas666) [reply src/middleware.ts]: I found one I missed.. committing it now

Comment (jaas666) [reply src/middleware.ts]: Done

**Review by nick-y-ito — APPROVED** [review]
@jaas666 
Thank you for your outstanding contribution! Your work has really helped us save time 😊 

Currently, we're not actively seeking new contributors, mainly due to our limited capacity to manage contributions. However, we'd be delighted to have you contribute after our upcoming first release coming in a few weeks!

**Comment by jaas666** [comment]
Glad I could help. Thanks.

**Pull request merged**